.golangci.yaml

```
@@ -0,0 +1,146 @@
+run:
+  timeout: 5m
```

```suggestion
  timeout: 10m
```
We have it set to 10 min in CAPI, not sure if we want to do the same here

---

CAPI has a large code base, and that's why they increased the default timeout. In case of CAAPH it takes 17 seconds for golangci to finish. So probably we can keep the timeout as-is for now.